nextflow_schema.json

```
@@ -81,17 +81,35 @@
                     "fa_icon": "fas fa-barcode",
                     "description": "Trim barcodes from the output sequences in the FastQ files from basecaller."
                 },
-                "gpu_device": {
+                "kit": {
                     "type": "string",
-                    "default": "auto",
-                    "description": "Device specified in GPU mode using '--device'.",
+                    "default": "",
+                    "description": "sequencing kit'.",
```

Rather minimal docs here, do they need to follow a particular pattern? Is there a particuarly fixed list of categirues that are requird?
```suggestion
                    "description": "sequencing kit.",
```

---

The pattern of the sequencing kits is not homogenous (in my opinion), so I think people need to refer to their sequencing run info for the exact kit name